### PR TITLE
PMM-8187 Operator version available

### PR DESCRIPTION
**Pull request by tiagomotasantos**
**What this PR does / why we need it**: Adds information about a new operator version being available and link to the release notes of the new version.

**Which issue(s) this PR fixes**: https://jira.percona.com/browse/PMM-8187

Screenshot 2021-07-02 at 17 51 03

### CODE REVIEW

File: public/app/percona/dbaas/components/Kubernetes/Kubernetes.hooks.ts [hunk 1]
```diff
@@ -81,10 +92,37 @@ export const useKubernetes = (): [Kubernetes[], DeleteKubernetesAction, AddKuber
...
-const toModelList = (response: KubernetesListAPI): Kubernetes[] => (response.kubernetes_clusters ?? []).map(toModel);
+const toModelList = (response: KubernetesListAPI, checkUpdateResponse: CheckOperatorUpdateAPI): Kubernetes[] =>
+  (response.kubernetes_clusters ?? []).map(toModel(checkUpdateResponse));
 
-const toModel = (response: KubernetesAPI): Kubernetes => ({
-  kubernetesClusterName: response.kubernetes_cluster_name,
-  operators: response.operators,
-  status: response.status as KubernetesClusterStatus,
+const toModel = (checkUpdateResponse: CheckOperatorUpdateAPI) => ({
+  kubernetes_cluster_name,
```
Comment (nicolalamacchia) [inline public/app/percona/dbaas/components/Kubernetes/Kubernetes.hooks.ts:99]: nit: I would consider aliasing this to camelCase

**Review by nicolalamacchia — DISMISSED** [review]

File: public/app/percona/dbaas/components/Kubernetes/OperatorStatusItem/KubernetesOperatorStatus/OperatorStatus/OperatorStatus.styles.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  `,
+  statusUnsupported: css`
+    background-color: ${palette.gray1};
+    label: unsupported;
+  `,
+  statusUnavailable: css`
+    background-color: ${colors.formSwitchBgActive};
+    label: unavailable;
+  `,
+  InstallLinkIcon: css`
```
Comment (fabio-silva) [inline public/app/percona/dbaas/components/Kubernetes/OperatorStatusItem/KubernetesOperatorStatus/OperatorStatus/OperatorStatus.styles.ts:34]: nit: is this title case intentional?

**Review by fabio-silva — DISMISSED** [review]

File: public/app/percona/dbaas/components/Kubernetes/OperatorStatusItem/KubernetesOperatorStatus/OperatorStatus/OperatorStatus.styles.ts (hunk 2 above)
Comment (tiagomotasantos) [reply public/app/percona/dbaas/components/Kubernetes/OperatorStatusItem/KubernetesOperatorStatus/OperatorStatus/OperatorStatus.styles.ts]: No. Fixed!

File: public/app/percona/dbaas/components/Kubernetes/Kubernetes.hooks.ts (hunk 1 above)
Comment (tiagomotasantos) [reply public/app/percona/dbaas/components/Kubernetes/Kubernetes.hooks.ts]: Done!

**Review by fabio-silva — APPROVED** [review]

**Pull request closed without merging**